packages/core/src/App/Components/Elements/NotificationMessage/notification.jsx

```diff
@@ -18,6 +18,14 @@ const Notification = ({ data, removeNotificationMessage }) => {
     const destroy = is_closed_by_user => {
         removeNotificationMessage(data);
 
+        if (data.should_show_again) {
+            const closed_toast_notifications = JSON.parse(localStorage.getItem('closed_toast_notifications')) ?? [];
+            if (!closed_toast_notifications.includes(data.key)) {
+                closed_toast_notifications.push(data.key);
+                localStorage.setItem('closed_toast_notifications', JSON.stringify(closed_toast_notifications));
```

Comment:
how is this different from the `notification_messages` that we are storing locally?

Reply:
i also wonder if thats does the same thing, let me try to use it 😬

Reply:
Its doesnt work using `notification_messages` because the toast message will shows-up again after notifications refreshed

Reply:
i think best to have a separate one to handle closed toast notifications tbh 👀